cmake/onnxruntime_unittests.cmake

```diff
@@ -713,11 +713,17 @@ if (onnxruntime_BUILD_SERVER)
       set_source_files_properties("${TEST_SRC_DIR}/server/unit_tests/executor_test.cc" PROPERTIES COMPILE_FLAGS -Wno-unused-parameter)
     endif()
   endif()
-
+  if (onnxruntime_USE_OPENVINO_BINARY)
```

Comment:
I thought onnxruntime_USE_OPENVINO_BINARY option was removed in the R3.1 update PR